tests/sentry/api/endpoints/test_project_ownership.py

```diff
@@ -132,16 +132,13 @@ def test_update_with_streamline_targeting(self):
         }
 
     def test_get(self):
```

Comment:
Refactored test from related [project_ownership](https://getsentry.atlassian.net/browse/WOR-2675) ticket to make it easier to read